doc/mgr/ansible.rst

```diff
@@ -41,25 +41,32 @@ Enable the Ansible orchestrator module and use it with the :ref:`CLI <orchestrat
 Configuration
 =============
 
+The external Ansible Runner Service uses TLS mutual authentication to allow clients to use the API.
+A client certificate and a key files should be provided by the Administrator of the Ansible Runner Service for each manager node.
+This files should be copied in each of the manager nodes with read access for the ceph user.
+The destination folder for this files and the name of the files must be the same always in all the manager nodes,
+althought the certificate/key content of this files logically will be different in each node.
+
 Configuration must be set once the module is enabled by first time.
 
 This can be done in one monitor node via the configuration key facility on a
 cluster-wide level (so they apply to all manager instances) as follows::
 
 
-    # ceph config set mgr mgr/ansible/server_addr <ip_address/server_name>
-    # ceph config set mgr mgr/ansible/server_port <port>
-    # ceph config set mgr mgr/ansible/username <username>
-    # ceph config set mgr mgr/ansible/password <password>
-    # ceph config set mgr mgr/ansible/verify_server <verify_server_value>
+    # ceph config set mgr mgr/ansible/server_url <ip_address/server_name>:<port>
```

Comment:
As this is not a full url as it excludes the scheme, there is no way to provide a way to specify an HTTPS url to the ARS.

Reply:
ARS only works with an https protocol. 
But agree with you that probably the setting name is not friendly, it is not an url, it is just the "authority" component of the URL ... 
Using "url_authority" is also  not going to be very understandable for final users ... i think ...
So ... what do you think about:
"server_location"?

Reply:
> ARS only works with an https protocol.

perfect! :+1:

Reply:
OK!

Reply:
A long history .. this started with the same that we have in the dashboard "server_address", "server_port". @jcsp  and @leseb suggested to have this settings mixed  in only one ... "server_url" ... now "server_location" seems to be better... "server" has the problem that you can undertand that only need the server name/ip ... not the port...
I think that it is ok with this last change